control-announcements/src/main/java/org/oskari/announcements/actions/AnnouncementsHandler.java

```diff
@@ -21,13 +23,17 @@ public class AnnouncementsHandler extends RestActionHandler {
     public void handleGet(ActionParameters params) throws ActionException {
 
         try {
-            JSONObject result = new JSONObject();
-            if (params.getHttpParam("all", false) && params.getUser().isAdmin()) {
-                result = service.getAdminAnnouncements();
+            List<Announcement> announcements;
+            if (params.getUser().isAdmin()) {
+                announcements = service.getAnnouncements();
             } else {
-                result = service.getAnnouncements();
+                announcements = service.getActiveAnnouncements();
+            }
+            JSONArray result = new JSONArray();
+            for (Announcement announcement : announcements) {
+                result.put(announcement.asJSON());
             }
```

Comment:
Could we use AnnouncementsHelper.writeJSON() instead and get rid off asJSON() in announcement?